tasks.js

```diff
@@ -1 +1,2 @@
-module.exports = require('./dist/src/tasks');
+// @flow
+module.exports = require('./dist/tasks');
```

Comment:
there's a trick, if you copy the `package.json` in the `dist` folder after the build, you can publish from there and avoid having these files polluting your root just to re-export bundled files

Reply:
clever!

Reply:
ah I guess you don't want to do it